global-api/importer/crosswalk/crosswalk_grid_importer.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 29 16:43:02 2023
+
+@author: maureenfonseca
+"""
+
+# import Crosswalk Labs grid data into database
+# >> python crosswalk_grid_importer.py --database-uri DB_URI
+
+
+import argparse
+from datetime import datetime
+import os
+from sqlalchemy import create_engine, MetaData, Table
+from tqdm import tqdm
+from utils import (
+    area_of_polygon,
+    bounding_coords,
+    polygon_from_coords,
+    get_crosswalk,
+    insert_record,
+    uuid_generate_v3,
+)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--database_uri",
+        help="database URI (e.g. postgresql://ccglobal:@localhost/ccglobal)",
+        default=os.environ.get("DB_URI"),
+    )
+    args = parser.parse_args()
+
+    # load any file, just want the grid
+    ds = get_crosswalk('US', 'airport', 'hi')
+
+    engine = create_engine(args.database_uri)
+    metadata_obj = MetaData()
+
+    table = Table("crosswalk_GridCell", metadata_obj, autoload_with=engine)
+
+    lats = ds.lat.values
+    lons = ds.lon.values
+    
```

Comment:
This can be made more efficient. One solution is do something like this:

```python
lats = ds.lat.values.flatten()
lons = ds.lon.values.flatten()

assert len(lats) == len(lons), 'len of lats and lons do not match'

for lat, lon in zip(lats, lons):
    # do something with lat and lon
```

I'm "flattening" the arrays so they're 1D and, as a sanity check, making sure the lengths are the same.  With this solution we're only looping through the coordinates once.